File: FCUBEAPI/FleetTrans/Models/VehicleEngagementRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class VehicleEngagementRptListModel
    {
        public List<VehicleEngagementRptModel> VehicleEngagementRptList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/vehicleengagementrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Vehicleengagementrptmodel } from "./vehicleengagementrptmodel";

export class Vehicleengagementrptlistmodel {
    vehicleEngagementRptList: Vehicleengagementrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
